modules/tinymce/src/core/test/ts/browser/ModeTest.ts

```diff
@@ -108,4 +109,14 @@ describe('browser.tinymce.core.ModeTest', () => {
     assertMode(editor, 'readonly');
     assertBodyClass(editor, 'mce-content-readonly', true);
   });
+
+  it('TINY-6800: even in readonly mode copy event should be dispatched', () => {
+    const editor = hook.editor();
+
+    let copyEventCount = 0;
+    editor.on('copy', () => copyEventCount++);
```

Comment:
This isn't unbound at the end of the test and we should be cleaning up after ourselves.

Reply:
done